ajax/libs/antimoderate/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "antimoderate",
   "version": "0.0.2",
-  "description": "The progressive image loading library for great good!",
-  "filename": "antimoderate.min.js",
+  "description": "The progressive image loading library",
```

Comment:
It's fine to use origin description.
This is the same with https://github.com/whackashoe/antimoderate#antimoderate
Thanks.